### PR TITLE
Update Guide to Oppia Pages

### PR DESCRIPTION
**Pull request by U8NWXD**
This also creates a style guide for the wiki, which is a work in progress. I'll add to the style guide as I go.

### CODE REVIEW

**Comment by DubeySandeep** [comment]
@U8NWXD can we reduce the size of the images? I see out of images of size 700+kbs, have you tried compressing the images?

**Comment by DubeySandeep** [comment]
Also, please add TOC on top of the wiki page! (In general this wiki page looks good to me!)

**Comment by U8NWXD** [comment]
@DubeySandeep I've added a TOC and compressed the images. PTAL

File: How-to-access-Oppia-webpages.md
Comment (DubeySandeep) [inline How-to-access-Oppia-webpages.md:32]: Should we divide these into the following sections:
1. Admin pages
    - Super admin page
    - Contributor dashboard admin page
    - Release coordinator page 
2. User account pages
    - Creating a user account
    - Creating admin account
    - Delete account page
    - Preferences page
    - Profile page
3. Editor pages
    - ...
4. Player pages
5. Contributor pages
6. Static pages

File: How-to-access-Oppia-webpages.md
Comment (DubeySandeep) [inline How-to-access-Oppia-webpages.md:51]: We are removing the `administrator role` in PR https://github.com/oppia/oppia/pull/12989 [About to merge] After these changes we will just have super_admin. Note: we also have multiple sub-admin roles: Translation admin, Voiceover admin, Questions admin etc.

File: How-to-access-Oppia-webpages.md
Comment (DubeySandeep) [inline How-to-access-Oppia-webpages.md:118]: We are going to have a multi-role system where user have-to-have the full user role, over here it will be "add role".

**Review by DubeySandeep — CHANGES_REQUESTED** [review]
LGTM, left a  few minor comments PTAL!

**Comment by DubeySandeep** [comment]
Also, should we add to steps for access new pages which are added recently?

1. Release coordinator
2. Blog
3. Contributor dashboard admin
4. Blog admin

File: How-to-access-Oppia-webpages.md
Comment (U8NWXD) [reply How-to-access-Oppia-webpages.md:51]: The PR has failing backend tests, so this PR might be ready to merge first. How about we merge this PR and then update the docs once oppia/oppia#12989 merges? If oppia/oppia#12989 merges first, then we can update this PR before merging

File: How-to-access-Oppia-webpages.md
Comment (U8NWXD) [reply How-to-access-Oppia-webpages.md:118]: Like in my response comment above, I think we should only update the docs after the code has changed

File: How-to-access-Oppia-webpages.md
Comment (U8NWXD) [reply How-to-access-Oppia-webpages.md]: done (though I used a slightly different grouping)

**Comment by U8NWXD** [comment]
@DubeySandeep I've addressed your comments. PTAL

File: How-to-access-Oppia-webpages.md
Comment (DubeySandeep) [reply How-to-access-Oppia-webpages.md:59]: The PR got merged yesterday! Let's update this page :)

File: How-to-access-Oppia-webpages.md
Comment (DubeySandeep) [reply How-to-access-Oppia-webpages.md:198]: DItto as above!

**Comment by DubeySandeep** [comment]
@U8NWXD have we add all the pages mentioned [here](https://github.com/U8NWXD/oppia-wiki/pull/7#issuecomment-886007026)? Let me know if you need any info about those pages?

File: How-to-access-Oppia-webpages.md
Comment (U8NWXD) [reply How-to-access-Oppia-webpages.md]: done

File: How-to-access-Oppia-webpages.md
Comment (U8NWXD) [reply How-to-access-Oppia-webpages.md]: done

**Comment by U8NWXD** [comment]
@DubeySandeep PTAL. I've added the release coordinator and contributor dashboard admin pages. We decided to wait on the blog pages until they're working

**Review by DubeySandeep — APPROVED** [review]
LGTM!

@seanlip Can you take a final pass?

File: Wiki-style-guide.md
Comment (seanlip) [inline Wiki-style-guide.md:5]: Maybe give examples inline for what the code/display for these looks like (markdown headers, markdown block quotes, etc.).

If possible, try to also explain/show what not to do.

File: Wiki-style-guide.md
Comment (seanlip) [inline Wiki-style-guide.md:10]: Link to a definition of sentence case / title case, or give examples.

In general, explain jargon.

File: Wiki-style-guide.md
Comment (seanlip) [inline Wiki-style-guide.md:11]: I think punctuation is probably worth calling out as its own, separate point.

File: Wiki-style-guide.md
Comment (U8NWXD) [reply Wiki-style-guide.md:27]: done

File: Wiki-style-guide.md
Comment (U8NWXD) [reply Wiki-style-guide.md]: done

File: Wiki-style-guide.md
Comment (U8NWXD) [reply Wiki-style-guide.md]: done

**Pull request merged**
